src/sentry_sdk.cpp

```diff
@@ -96,7 +99,13 @@ void SentrySDK::set_user(const Ref<SentryUser> &p_user) {
 	}
 }
 
+Ref<SentryUser> SentrySDK::get_user() const {
+	MutexLock lock(*user_mutex.ptr());
```

Comment:
I assume ppl would use this method to mutate the user. now they will have a clone. not sure this is the way to go